modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSortAggregateTest.java

```diff
@@ -67,8 +70,118 @@ public void mapReduceAggregate() {
+
+            assertEquals(1, res.size());
+            assertEquals(List.of(7L, 5L), res.get(0));
+
+            res = sql(appendDisabledRules("SELECT COUNT(a) as a, s FROM testMe GROUP BY s ORDER BY a, s", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(4, res.size());
+            assertEquals(List.of(List.of(1L, "ahello"), Arrays.asList(1L, null), List.of(2L, "hello"), List.of(3L, "world")), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(a) as a, AVG(a) as b, MIN(a), MIN(b), s FROM testMe GROUP BY s ORDER BY a, b",
+                    disabledRules));
+            res0 = transformResults(res);
+            assertEquals(4, res.size());
+            assertEquals(List.of(List.of(1L, 10, 10, 5, "ahello"), Arrays.asList(1L, 11, 11, 3, null), List.of(2L, 11, 11, 1, "hello"),
+                    List.of(3L, 12, 12, 2, "world")), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(a) as a, AVG(a) as bb, MIN(a), MIN(b), s FROM testMe GROUP BY s, b ORDER BY a, s",
+                    disabledRules));
+            res0 = transformResults(res);
+            assertEquals(6, res.size());
+            assertEquals(List.of(List.of(1L, 10, 10, 5, "ahello"), List.of(1L, 11, 11, 1, "hello"), List.of(1L, 11, 11, 3, "hello"),
+                    List.of(1L, 13, 13, 6, "world"), Arrays.asList(1L, 11, 11, 3, null), List.of(2L, 12, 12, 2, "world")), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(a) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(7L), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(DISTINCT(a)) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(4L), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(a), COUNT(s), COUNT(*) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(7L, 6L, 7L), res0.get(0));
+
+            res = sql(appendDisabledRules("SELECT AVG(a) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(11), res0);
+
+            res = sql(appendDisabledRules("SELECT MIN(a) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(10), res0);
+
+            res = sql(appendDisabledRules("SELECT COUNT(a), COUNT(DISTINCT(a)) FROM testMe", disabledRules));
+            res0 = transformResults(res);
+            assertEquals(1, res.size());
+            assertEquals(List.of(7L, 4L), res0.get(0));
+        } finally {
+            sql("DROP TABLE testMe");
+        }
+    }
+
+    private String appendDisabledRules(String sql, String rules) {
```

Comment:
Maybe, it would be better to extend parent `sql` method with disabled rules instead of this one. WDYT?
```
   String sql(String sql, String... rules) {
```
or
```
    private String appendDisabledRules(String sql, List<String> rules) {
```

Reply:
I understand but seems that check output results with **sql()** is not comfortable as with QueryChecker (which need to be rafactored a bit, issue is linked above) let`s do it in checker ?